web-react/src/auth/RoleView.jsx

```diff
@@ -4,7 +4,7 @@ import { MemberContext } from '../contexts/MemberContext'
 import useAuth from './useAuth'
 
 const RoleView = (props) => {
-  const { roles, children, verifyId, permittedStream } = props
+  const { roles, children, verifyId, permittedStream, noIncome } = props
```

Comment:
There's no need to pass `noIncome` as a prop since it already exists as a property on currentUser

Reply:
ohhh cool, and tbh i don't use RoleView anywhere so i think this change is quite redundant